projects/proposals/K8s on LinuxOne.md

```diff
@@ -0,0 +1,113 @@
+* Name of project (must be unique within OMP)
+
+Kubernetes on LinuxOne
```

Comment:
We should probably drop the 'LinuxOne' term and use 's390x' instead to avoid product names.

Reply:
Kubernetes already builds on s390x. What would the real purpose of this project be?

Reply:
> We should probably drop the 'LinuxOne' term and use 's390x' instead to avoid product names.

Sure. I thought the same. I will make the changes.


> Kubernetes already builds on s390x. What would the real purpose of this project be?

Only the Kubernetes binaries are supported for s390x. But the network plugins, e2e tests and other plugins are not supported as of now. Check https://github.com/kubernetes/examples/pull/318 and  https://k8s-testgrid.appspot.com/sig-node-ppc64le#conformance. This requires building and maintaining different Dockerfiles.

Reply:
@markkp Does @rajula96reddy's response answer your question?

Reply:
> We should probably drop the 'LinuxOne' term and use 's390x' instead to avoid product names.

zLinux is another industry term that is not a product but a reference to Linux on Z hardware

Reply:
Many of our projects have gone the route of a codename that alludes to the focus of the project ( for example, Feilong for the the previous z/VM Cloud Connector work ). That might be another good option - which at the same time opens up the possibility of a cool looking logo LOL.

We can work more on the project name during incubation if the TSC is fine with that.

Reply:
> @markkp Does @rajula96reddy's response answer your question?

Yes.